tests/test_pysoundfile.py

```diff
@@ -660,7 +665,7 @@ def test_seek_in_rplus_mode(sf_stereo_rplus):
 
 @pytest.mark.parametrize("use_default", [True, False])
 def test_truncate(file_stereo_rplus, use_default):
-    if isinstance(file_stereo_rplus, (str, int)):
+    if isinstance(file_stereo_rplus, (str, int)) or hasattr(file_stereo_rplus, '__fspath__'):
```

Comment:
E501 line too long (93 > 79 characters)

Reply:
I actually left that in there because the second line would line up with the `with` in the next line, which looks confusing.

I've tried a few other combinations, like breaking after the comma in `hasattr`, but that doesn't look any better, either.

That's why I opted for the longer line.

Reply:
The broken line should of course not align with the following line, this would cause another `pycodestyle` warning:

    E129 visually indented line with same indent as next logical line

`autopep8` suggests this:

```python
def test_truncate(file_stereo_rplus, use_default):
    if isinstance(file_stereo_rplus, (str, int)) or hasattr(
            file_stereo_rplus, '__fspath__'):
        with sf.SoundFile(file_stereo_rplus, 'r+', closefd=False) as f:
```

This doesn't look particularly beautiful, but IMHO it's better than violating PEP8 in this case.

Reply:
I highly recommend reading PEP8, in this case the section about indentation (https://www.python.org/dev/peps/pep-0008/#indentation)  tells us a few interesting things:

> The 4-space rule is optional for continuation lines.

> This PEP takes no explicit position on how (or whether) to further visually distinguish such conditional lines from the nested suite inside the `if`-statement.

... and as one example it use it uses an additional 4 spaces of indentation.

Interestingly, PEP8 sometimes changes its opinion. When I was re-reading the abovementioned section, I realized that the recommended position of operators at line breaks has changed since I last read PEP8: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator (here's the change: https://github.com/python/peps/commit/c59c4376ad233a62ca4b3a6060c81368bd21e85b). IMHO that's an improvement!